test/integration/test-example-validation.js

```diff
@@ -30,7 +30,7 @@ if (!window.validatorLoad) {
   })();
 }
 
-describe.configure().retryOnSaucelabs().run('example', function() {
+describe.skip('example', function() {
```

Comment:
@aghassemi, [here](https://github.com/ampproject/amphtml/blob/master/build-system/pr-check.js#L253) is where the unit tests are run for PRs and on master. We used to run using old chrome on saucelabs, but no longer do.

I didn't think we ran integration tests with `gulp test`, unless the `--integration` arg was passed in, so your comment above comes as a surprise to me.